bin/mv.js

```diff
@@ -0,0 +1,239 @@
+
+module.exports = Cli.createCommand('mv', {
+    description: 'Move a named webtask',
+    plugins: [
+        require('./_plugins/profile')
+    ],
+    options: {
+        'target-container': {
+            description: 'Target container',
+            type: 'string',
+            dest: 'targetContainer'
+        },
+        'target-profile': {
+            description: 'Target profile',
+            type: 'string',
+            dest: 'targetProfile'
+        },
+    },
+    params: {
+        'source': {
+            description: 'Source webtask name',
+            type: 'string',
+            required: true,
+        },
+        'target': {
+            description: 'Target webtask name',
+            type: 'string',
+            required: false,
+        },
+    },
+    handler: handleWebtaskMove,
+});
+
+function handleWebtaskMove(args) {
+    let options = _(args).pick(['targetContainer', 'targetProfile']).omitBy(_.isNull).value();
+    let targetName = args.target || args.source;
+
+    return moveWebtask(args.profile, args.source, {
+        profile: options.targetProfile,
+        container: options.targetContainer,
+        name: targetName
+    }).then(function () {
+        console.log(chalk.green('Moved webtask: %s'), chalk.bold(targetName));
+    });
+}
+
+function moveWebtask(profile, name, target) {
+    debug('moveWebtask: profile=%j, name=%s, target=%j',
+        _.omit(profile, 'token'), name, target);
+
+    let sourceWebtask;
+
+    if (equal({name: name, container: profile.container, profile: profile.name}, target)) {
+        throw Cli.error.invalid('Webtasks are identical. Use a different target name, container or profile.');
+    }
+
+    return profile.getWebtask({name: name})
+        .catch(function (err) {
+            if (err.statusCode === 404) {
+                throw Cli.error.notFound('No such webtask: ' + chalk.bold(name));
+            }
+            throw err;
+        })
+        .then(function (webtask) {
+            sourceWebtask = webtask;
+            return webtask.inspect({
+                decrypt: true,
+                fetch_code: true
+            });
+        })
+        .then(function (data) {
+            return copy(sourceWebtask, data, target);
+        })
+        .then(function () {
+            return sourceWebtask.remove();
+        });
+}
+
+function equal(sourceParams, targetParams) {
+    return _.isEqual(sourceParams, {
+        name: targetParams.name,
+        container: targetParams.container || sourceParams.container,
+        profile: targetParams.profile || sourceParams.profile
+    });
+}
+
+function copy(webtask, data, target) {
+    debug('copy: webtask=%j, data=%j, target=%j',
+        _.omit(webtask, 'token'), data, target);
+
+    if (!data.jtn) {
+        throw Cli.error.cancelled('Not a named webtask.');
+    }
+
+    let claims = cloneWebtaskData(data);
+    claims.jtn = target.name || claims.jtn;
+    claims.ten = target.container || claims.ten;
```

Comment:
I believe we should always be using `claims.ten` here. In a general case this can be a regex that is broader than just `target.container`.